virttest/utils_disk.py

```diff
@@ -87,24 +111,31 @@ def mount(src, dst, fstype=None, options=None, verbose=False):
         cmd.extend(['-o', options])
     cmd.extend([src, dst])
```

Comment:
What if no `src`, or `src` is a relative path?

Reply:
@luckyh I supposed this requires src must exist and must be a real path. Otherwise the case should fail with the command execution failure.

Reply:
ok, make sense, seems "no src" would not happen, and user can avoid to use relative path.